### ISSUE TITLE
Script for Wireguard interface metrics

### ISSUE THREAD

**Issue by Lol3rrr** [prometheus-community/node-exporter-textfile-collector-scripts#124]
# Motivation
Im using wireguard as a vpn on my server and would like to be able to easily load at least basic metrics for it into prometheus.

# Proposition
I would like to add a new script that exports metrics regarding wireguard interface metrics. This should be possible by using the `wg show all dump` command to get the Data to export some information about it

**Comment by dswarbrick** [comment]
Have you considered using a dedicated exporter such as https://github.com/MindFlavor/prometheus_wireguard_exporter?

**Comment by Lol3rrr** [comment]
Yeah, but considering that there aren't that many metrics I find it a bit overkill to add an entire extra exporter.

I also previously looked at the project you linked and because there are no pre-built binaries made it a hassle to deploy for my setup and the docker version just didn't work in my setup.
